pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
+
+- Read JSON stream into array and output:
+
+`jq -s '' {{file}}`
+
+- Output first element in JSON file:
+
+`jq '.[0]' {{file}}`
+
+- Output "key" of first element in JSON file:
+
+`jq '.[0].{{key}}' {{file}}`
+
+- Output "key" of each element in JSON file:
+
+`jq 'map(.{{key}})' {{file}}`
```

Comment:
This looks really useful. Perhaps adding an example that outputs multiple keys of a each object would be useful too? It's fine as is though.